packages/shared/src/components/squads/settings/SquadModerationSettingsSection.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { ReactElement, useState } from 'react';
+import { SquadSettingsSection } from './SquadSettingsSection';
+import { Radio } from '../../fields/Radio';
+import { SourceMemberRole } from '../../../graphql/sources';
+import { Switch } from '../../fields/Switch';
+import { WidgetCard } from '../../widgets/WidgetCard';
+import { SimpleTooltip } from '../../tooltips';
+import { useViewSize, ViewSize } from '../../../hooks/useViewSize';
+import { MiniCloseIcon } from '../../icons';
+
+interface PermissionSectionProps {
```

Comment:
We should maintain the name consistent, maybe rename into `SquadModerationSettingsSectionProps`?

Reply:
Agreed, I missed this one. Thanks!